papers/ReTraCk/requirements.txt

```diff
@@ -11,7 +11,7 @@ gevent
 # Dense Retriever
 pysolr==3.8.1
 emoji==0.5.3
-regex==2019.8.19
+#regex==2019.8.19
```

Comment:
And this one?

Reply:
It reports installation errors when specifying the regex==2019.8.19, but it can automatically search compatible regex version (2022.9.13 in my side).
So I think there is no need to specify the regex version.